backends/samsung/utils/export_utils.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2025 Samsung Electronics Co. LTD
+# All rights reserved
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List, Optional, Tuple
+
+import executorch.exir as exir
+import torch
+from executorch.exir import EdgeCompileConfig, ExportedProgram
+from executorch.exir.backend.backend_details import CompileSpec
+from executorch.exir.program._program import (
+    to_edge_transform_and_lower,
+)
+
+
+def to_edge_transform_and_lower_to_enn(
```

Comment:
Do you need this function? I know a few other backends have a similar wrapper, but it would be ideal to use the standard to_edge_transform_and_lower, if possible. If you have any functionality that requires a custom wrapper, we can definitely try to figure out how to extend the core APIs to facilitate that.